minicluster/src/main/java/org/apache/accumulo/cluster/standalone/StandaloneClusterControl.java

```diff
@@ -149,9 +141,8 @@ String getJarFromClass(Class<?> clz) {
 
   @Override
   public void adminStopAll() throws IOException {
-    File confDir = getConfDir();
-    String master = getHosts(new File(confDir, "masters")).get(0);
-    String[] cmd = new String[] {SUDO_CMD, "-u", user, ACCUMULO_CONF_DIR + serverAccumuloConfDir, accumuloPath, Admin.class.getName(), "stopAll"};
+    String master = getHosts(MASTER_HOSTS_FILE).get(0);
+    String[] cmd = new String[] {SUDO_CMD, "-u", user, accumuloPath, Admin.class.getName(), "stopAll"};
```

Comment:
Functionally, yes, I think that would work.

> However, I don't think it's hard at this point to add a solution for separate client & server configuration directories that avoid use of ACCUMULO_CONF_DIR.

Yeah, it doesn't matter too much to me the means, just that the functionality exists.

> ${conf} defaults to the conf directory in the Accumulo tarball installation

Ok, and that's extrapolated based on the assumption that the `bin/` dir is next to the `conf/` dir (or at least a symlink exists to enable that). Sounds fine.

As a general statement, I'm lamenting that task that this change will entail to update all of my downstream docs/code to make this work. This will be a very cross cutting change for me to have to make and will be a pain in the butt. Sadly, we don't have a good strategy for defining what compatibility is here (and what the 1.x to 2.x means).

FYI @busbey as this may also affect things on the CM side..